pkg/controller/migmigration/migmigration_controller.go

```diff
@@ -135,12 +135,12 @@ func (r *ReconcileMigMigration) Reconcile(request reconcile.Request) (reconcile.
 	}
 
 	// Completed.
-	if migration.Status.HasAnyCondition(Succeeded, Failed) {
+	if migration.Status.Phase == Completed {
```

Comment:
Does this diff back out the changes that moved the controller to use Conditions instead of phase to track completion? Is this a merge issue or intentional?